WordPress/Classes/Utility/WPError.m

```diff
@@ -88,7 +89,7 @@ + (void)showNetworkingAlertWithError:(NSError *)error title:(NSString *)title
     }
 
     if (message == nil) {
-        message = [error localizedDescription];
+        message = [[error localizedDescription] stringByStrippingHTML];
```

Comment:
I aimed at the method that receives the `NSError` instance, because it's the one that contains a string that comes from elsewhere (meaning that we do have control over whatever is fed over to `showAlertWithTitle:message:withSupportButton:okPressedBlock:`).

Nevertheless, for the sake of safety, let's move it over. Thanks!